metadeploy/api/jobs.py

```diff
@@ -296,3 +262,66 @@ def expire_preflights():
 
 
 expire_preflights_job = job(expire_preflights)
+
+
+def create_scratch_org(*, plan_id, email, org_name, result_id):
+    plan = Plan.objects.get(id=plan_id)
+    org = ScratchOrgJob.objects.get(id=result_id)
+    repo_url = plan.version.product.repo_url
+    repo_owner, repo_name = extract_user_and_repo(repo_url)
+    commit_ish = plan.commit_ish
+    try:
+        with local_github_checkout(
+            repo_owner, repo_name, commit_ish=commit_ish
+        ) as repo_root:
+            scratch_org_config, _, org_config = create_scratch_org_on_sf(
```

Comment:
@davisagli I'm not really clear on the difference between the `scratch_org_config` and the `org_config`, or what is in either, so everything that uses them is guesswork. Your input would be appreciated.